### DISCUSSION TITLE
sunshine doesn't remember PIN for clients, sunshine.conf 0 bytes

### DISCUSSION THREAD

**Discussion by perler** [LizardByte/community#214] [Sunshine]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Is your issue described in the documentation?

- [X] I have read the documentation

### Is your issue present in the nightly release?

- [X] This issue is present in the nightly release

### Describe the Bug

After the first successful authentiction of a client via PIN further connections should happen without the need to enter a PIN again. This is not happening here. After shutting down and restarting the sunshine server, the server is locked again.

What I see is, that  sunshine.conf and  sunshine.log are empty/0 bytes.  

### Expected Behavior

after first authentication, clients should connect without entering PIN.

### Host Operating System

Windows

### Operating System Version

windows 10 22h2

### Architecture

64 bit

### Sunshine commit or version

0.19.1.d70d084f9fbb4e0150977a89d94937418a3ccf9c

### Package

Windows - installer

### GPU Type

Nvidia

### GPU Model

RTX 3060

### GPU Driver/Mesa Version

531.41

### Config

```Shell
n/a s. above
```

### Relevant log output

```shell
n/a s. above
```

**Comment by ReenigneArcher** [comment]
`sunshine.conf` is supposed to be empty unless you modified any config settings in the UI.

**Comment by perler** [comment]
ok, so that's not the reason why the PIN is not remembered.

I did some more tests and the problem is a bit different than on the first look:

You can login to the host without re-entering the PIN if you completely shutdown the host between sessions. The host in question was usually put into sleep mode between sessions and in this case, after resuming the host, it is viewed as "not available" ❗ on a windows client and as "locked" 🔓 on an Apple TV. After completely shutting down the host and starting again the host is accessible again from both clients.

**Reply by ReenigneArcher** [reply]
That just indicates that Sunshine is not running.

[a comment by an automated account is not shown]
